pkg/providers/ovirt/validation/validators/validator-wrapper.go

```diff
@@ -23,12 +28,17 @@ func NewValidatorWrapper(client client.Client) *ValidatorWrapper {
 	return &ValidatorWrapper{
 		networkMappingValidator: NewNetworkMappingValidator(&netAttachDefProvider),
 		storageMappingValidator: NewStorageMappingValidator(&storageClassesProvider),
+		kvConfigProvider:        kvConfigProvider,
 	}
 }
 
 // ValidateVM wraps validators package implementation of ValidateVM function
 func (v *ValidatorWrapper) ValidateVM(vm *ovirtsdk.Vm) []ValidationFailure {
-	return ValidateVM(vm)
+	kvConfig, err := v.kvConfigProvider.GetConfig()
+	if err != nil {
+		logger.Error(err, "Cannot get KubeVirt cluster config.")
+	}
+	return ValidateVM(vm, kvConfig)
```

Comment:
What would be `kvConfig` value when we get an `err` while getting it? I am especially interested in `LiveMigrationEnabled` because we use it even we have an issue to get it.

Reply:
It will be initialized with zero values and will claim there is no support for live migration. 
IMO that's optimal approach to handling that error here; if we propagated the error up the call stack, we would always re-queue the request and in current implementation we will re-queue it only if live migration feature gate state would be important (source vm has configuration requiring it and we fail validation due to assumed lack of support for it).
What's your opinion?

Reply:
I am not sure. Having no kubevirt config seems critical and means that something is not right.

Reply:
The KubeVirt configmap is optional and in some regular cases we might not have it; the behaviour of the config provider will be different - it won't return error but the returned config will be the same.